tests/data/ingress-mtls/policies/ingress-mtls-invalid.yaml

```diff
@@ -3,7 +3,7 @@ kind: Policy
 metadata:
   name: ingress-mtls-policy
 spec:
-  ingredsssMTLS: # invalid/mis-spelled yaml keys
-    clientCertSecrettt: ingress-mtls-secret
+  ingress_MTLS: # invalid/mis-spelled yaml keys
+    clientCertSecret_: ingress-mtls-secret
```

Comment:
I understand you're trying to misspell things, but there are ways to generate invalid content that doesn't result in spell checkers screaming. As you can see, there are plenty of misspelled items, some of which are user facing. I argue it's better to avoid misspelling words when possible, and using a `_` should work for that purpose. (If not `_`, then adding another word or swapping the order of words or substituting a word...)